### PR TITLE
Add doc to config.sample for enforce LDAP home folder naming rule

### PR DESCRIPTION
**Pull request by MorrisJobke**
- forgotten to be added in #16891

cc @Xenopathic @carlaschroder for wording fixes

cc @blizzz for content

### CODE REVIEW

**Comment by blizzz** [comment]
:+1:

File: config/config.sample.php [hunk 1]
```diff
@@ -806,6 +806,21 @@
  */
 'ldapUserCleanupInterval' => 51,
 
+/**
+ * Enforce the existance of the home folder naming rule for all users
```
Comment (Xenopathic) [inline config/config.sample.php]: existence

**Comment by Xenopathic** [comment]
Other than the spelling fix, the doc makes sense :+1:

**Comment by PVince81** [comment]
@DeepDiver1975 please merge this trivial addition

**Comment by MorrisJobke** [comment]
If fixed the typo

**Pull request merged**